### ISSUE TITLE
different behavior angular-animate.js angular-animate.min.js (1.2.26) in Firefox 33.0.2

### ISSUE THREAD

**Issue by pepperblue** [angular/angular.js#9936]
I use SolydK 64bit with latest Firefox (33.0.2). The following code behaves different when using minified version of angular-animate:

----------- html ------------------
```
<!DOCTYPE html>
<html lang="de" xmlns="http://www.w3.org/1999/html">
<head>
    <meta charset="UTF-8">
    <title>AnimateTest</title>
    <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.2.26/angular.min.js"></script>
    <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.2.26/angular-animate.js"></script>
    <link href="main.css" rel="stylesheet" type="text/css">
</head>
<body data-ng-app="ngAnimate">
    <div class="animate-show" data-ng-hide="showPartial">
        Some content here ...
    </div>
    <input type="checkbox" data-ng-model="showPartial">Click here ...
</body>
</html>
```

----------- css --------------------
```
.animate-show {
    -webkit-transition:all linear 0.5s;
    transition:all linear 0.5s;
    line-height:20px;
    opacity:1;
    padding:10px;
    border:1px solid black;
    background:white;
}

.animate-show.ng-hide {
    line-height:0;
    opacity:0;
    padding:0 10px;
}

.check-element {
    padding:10px;
    border:1px solid black;
    background:white;
}
```

------------------------------------

Using the unminified version of angular-animate everything works fine. Using the minified version hiding the `<div>` is animated but showing is not (it just pops out). In Chromium everything works as expected so it could be a Firefox-Bug, but with Angular Version 1.3.1 things are working in Firefox too ... so it should be an issue you can solve in the 'stable' branch by yourself ...

**Comment by matsko** [comment]
Good things come from patience and hard work: https://github.com/angular/angular.js/pull/11469

**Comment by matsko** [comment]
This fix touches 1.2, 1.3 and the 1.4 beta.
